From: Sjoerd Simons <sjoerd@collabora.co.uk>
Date: Mon, 20 Jan 2020 16:04:16 +0100
Subject: Disable tests that depend on grub

Apertis doesn't ship grub; so disable tests that need grub-common to be
installed
---
 test/rauc.t | 64 ++++++++++++++++++++++++++++++-------------------------------
 1 file changed, 32 insertions(+), 32 deletions(-)

diff --git a/test/rauc.t b/test/rauc.t
index aa3873a..40302dd 100755
--- a/test/rauc.t
+++ b/test/rauc.t
@@ -372,38 +372,38 @@ test_expect_success !SERVICE "rauc status mark-active: internally" "
   rauc -c $SHARNESS_TEST_DIRECTORY/test.conf --override-boot-slot=system0 status mark-active
 "
 
-test_expect_success SERVICE "rauc status mark-good: via D-Bus" "
-  start_rauc_dbus_service \
-    --conf=${SHARNESS_TEST_DIRECTORY}/test.conf \
-    --override-boot-slot=system1 &&
-  test_when_finished stop_rauc_dbus_service &&
-  rauc \
-    --conf=${SHARNESS_TEST_DIRECTORY}/test.conf \
-    --override-boot-slot=system1 \
-    status mark-good
-"
-
-test_expect_success SERVICE "rauc status mark-bad: via D-Bus" "
-  start_rauc_dbus_service \
-    --conf=${SHARNESS_TEST_DIRECTORY}/test.conf \
-    --override-boot-slot=system1 &&
-  test_when_finished stop_rauc_dbus_service &&
-  rauc \
-    --conf=${SHARNESS_TEST_DIRECTORY}/test.conf \
-    --override-boot-slot=system1 \
-    status mark-bad
-"
-
-test_expect_success SERVICE "rauc status mark-active: via D-Bus" "
-  start_rauc_dbus_service \
-    --conf=${SHARNESS_TEST_DIRECTORY}/test.conf \
-    --override-boot-slot=system1 &&
-  test_when_finished stop_rauc_dbus_service &&
-  rauc \
-    --conf=${SHARNESS_TEST_DIRECTORY}/test.conf \
-    --override-boot-slot=system1 \
-    status mark-active
-"
+#test_expect_success SERVICE "rauc status mark-good: via D-Bus" "
+#  start_rauc_dbus_service \
+#    --conf=${SHARNESS_TEST_DIRECTORY}/test.conf \
+#    --override-boot-slot=system1 &&
+#  test_when_finished stop_rauc_dbus_service &&
+#  rauc \
+#    --conf=${SHARNESS_TEST_DIRECTORY}/test.conf \
+#    --override-boot-slot=system1 \
+#    status mark-good
+#"
+
+#test_expect_success SERVICE "rauc status mark-bad: via D-Bus" "
+#  start_rauc_dbus_service \
+#    --conf=${SHARNESS_TEST_DIRECTORY}/test.conf \
+#    --override-boot-slot=system1 &&
+#  test_when_finished stop_rauc_dbus_service &&
+#  rauc \
+#    --conf=${SHARNESS_TEST_DIRECTORY}/test.conf \
+#    --override-boot-slot=system1 \
+#    status mark-bad
+#"
+
+#test_expect_success SERVICE "rauc status mark-active: via D-Bus" "
+#  start_rauc_dbus_service \
+#    --conf=${SHARNESS_TEST_DIRECTORY}/test.conf \
+#    --override-boot-slot=system1 &&
+#  test_when_finished stop_rauc_dbus_service &&
+#  rauc \
+#    --conf=${SHARNESS_TEST_DIRECTORY}/test.conf \
+#    --override-boot-slot=system1 \
+#    status mark-active
+#"
 
 test_expect_success "rauc install invalid local paths" "
   test_must_fail rauc install foo &&
